app/services/mapping/governmentagencygoodsitem/GovernmentAgencyGoodsItemBuilder.scala

```diff
@@ -37,6 +37,7 @@ object GovernmentAgencyGoodsItemBuilder {
       .toList
       .asJava
 
+  //scalastyle:off method.length
```

Comment:
:(

Reply:
Yeah, I know, but it's better to read file that it's not underlined :( we need to solve and avoid really long methods